### ISSUE TITLE
Connection time out error - vmware, Windows 8

### ISSUE THREAD

**Issue by squaredindex** [puphpet/puphpet#1541]
I'm getting **ERR_CONNECTION_TIMED_OUT** on both **192.168.57.101** and my domain (charthamvh.dev) -
![timeout](https://cloud.githubusercontent.com/assets/7317110/7220171/25129596-e6b6-11e4-914f-1035de7f4d60.png)

I'm running on **Windows 8** and using **VMWare Workstation 11**. My VM is using **centos65-x64**. I'm also using **Nginx** not apache.

#### My hosts file is configured correctly, as far as I can see:
![hosts](https://cloud.githubusercontent.com/assets/7317110/7220206/10f2399e-e6b7-11e4-8ec3-34b8e7cc6387.png)

### SSH
If I ssh in I can see my files, but they just won't load in the browser for some reason :disappointed:  
![ssh](https://cloud.githubusercontent.com/assets/7317110/7220458/a93bc6ae-e6bf-11e4-94e8-c4f44439ac2c.png)

# Here's my config.yaml:

```yaml
vagrantfile:
    target: local
    vm:
        box: puphpet/centos65-x64
        box_url: puphpet/centos65-x64
        hostname: ''
        memory: '512'
        cpus: '1'
        chosen_provider: vmware_workstation
        network:
            private_network: 192.168.57.101
            forwarded_port:
                vflnp_4wxrvog12obc:
                    host: '7741'
                    guest: '22'
        post_up_message: ''
        provider:
            virtualbox:
                modifyvm:
                    natdnshostresolver1: on
            vmware:
                numvcpus: 1
            parallels:
                cpus: 1
        provision:
            puppet:
                manifests_path: puphpet/puppet
                manifest_file: site.pp
                module_path: puphpet/puppet/modules
                options:
                    - '--verbose'
                    - '--hiera_config /vagrant/puphpet/puppet/hiera.yaml'
                    - '--parser future'
        synced_folder:
            vflsf_4qmeziddsqn1:
                source: ./public/
                target: /var/www
                sync_type: default
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
        usable_port_range:
            start: 10200
            stop: 10500
    ssh:
        host: null
        port: null
        private_key_path: null
        username: vagrant
        guest_port: null
        keep_alive: true
        forward_agent: false
        forward_x11: false
        shell: 'bash -l'
    vagrant:
        host: detect
server:
    install: '1'
    packages:
        - vim-common
users_groups:
    install: '1'
    groups: {  }
    users: {  }
firewall:
    install: '1'
    rules: {  }
cron:
    install: '1'
    jobs: {  }
nginx:
    install: '1'
    settings:
        default_vhost: 1
        proxy_buffer_size: 128k
        proxy_buffers: '4 256k'
    upstreams: {  }
    vhosts:
        nxv_ej2m13zf4aq5:
            server_name: charthamvh.dev
            server_aliases:
                - www.charthamvh.dev
            www_root: /var/www/public
            listen_port: '80'
            index_files:
                - index.html
                - index.htm
                - index.php
            client_max_body_size: 1m
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_port: '443'
            rewrite_to_https: '1'
            spdy: '1'
            locations:
                nxvl_16lntvugfkyf:
                    location: /
                    autoindex: off
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: ''
                    fastcgi_index: ''
                    fastcgi_split_path: ''
                nxvl_1hpcf2d6wsq5:
                    location: '~ \.php$'
                    autoindex: off
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: '127.0.0.1:9000'
                    fastcgi_index: index.php
                    fastcgi_split_path: '^(.+\.php)(/.*)$'
                    fast_cgi_params_extra:
                        - 'SCRIPT_FILENAME $request_filename'
                        - 'APP_ENV dev'
    proxies: {  }
apache:
    install: '0'
    settings:
        user: www-data
        group: www-data
        default_vhost: true
        manage_user: false
        manage_group: false
        sendfile: 0
    modules:
        - proxy_fcgi
        - rewrite
    vhosts:
        av_okpytqwk067d:
            servername: awesome.dev
            serveraliases:
                - www.awesome.dev
            docroot: /var/www/awesome
            port: '80'
            setenv:
                - 'APP_ENV dev'
            custom_fragment: ''
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            directories:
                avd_kxpntng0yaux:
                    path: /var/www/awesome
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_ao5hqyx3xhxx:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9000'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory
php:
    install: '1'
    settings:
        version: '56'
    fpm_settings:
        port: 9000
    composer: '1'
    composer_home: ''
    modules:
        php:
            - cli
            - intl
            - mcrypt
            - curl
            - imagick
            - gd
        pear: {  }
        pecl: {  }
    ini:
        display_errors: On
        error_reporting: '-1'
        session.save_path: /var/lib/php/session
        date.timezone: UTC
xdebug:
    install: '0'
    settings:
        xdebug.default_enable: '1'
        xdebug.remote_autostart: '0'
        xdebug.remote_connect_back: '1'
        xdebug.remote_enable: '1'
        xdebug.remote_handler: dbgp
        xdebug.remote_port: '9000'
xhprof:
    install: '0'
wpcli:
    install: '0'
    version: 6.3.0
drush:
    install: '0'
    version: null
ruby:
    install: '1'
    versions: {  }
python:
    install: '1'
    packages: {  }
    versions: {  }
nodejs:
    install: '0'
    npm_packages: {  }
hhvm:
    install: '0'
    nightly: 0
    composer: '1'
    composer_home: ''
    settings:
        host: 127.0.0.1
        port: '9000'
    ini:
        display_errors: On
        error_reporting: '-1'
    timezone: null
mysql:
    install: '1'
    settings:
        version: '5.6'
        root_password: '123'
        override_options: {  }
    adminer: '1'
    users:
        mysqlnu_dviag3c9hy0d:
            name: root
            password: '123'
    databases:
        mysqlnd_xwldehipvnzh:
            name: root
            sql: '123'
    grants:
        mysqlng_dcot11qgqqzl:
            user: dbuser
            table: '*.*'
            privileges:
                - ALL
postgresql:
    install: '0'
    settings:
        global:
            encoding: UTF8
            version: '9.3'
        server:
            postgres_password: '123'
    databases: {  }
    users: {  }
    grants: {  }
    adminer: 0
mongodb:
    install: '0'
    settings:
        auth: 1
        bind_ip: 127.0.0.1
        port: '27017'
    databases: {  }
redis:
    install: '0'
    settings:
        conf_port: '6379'
sqlite:
    install: '0'
    adminer: 0
    databases: {  }
mailcatcher:
    install: '0'
    settings:
        smtp_ip: 0.0.0.0
        smtp_port: 1025
        http_ip: 0.0.0.0
        http_port: '1080'
        mailcatcher_path: /usr/local/rvm/wrappers/default
        from_email_method: inline
beanstalkd:
    install: '0'
    settings:
        listenaddress: 0.0.0.0
        listenport: '13000'
        maxjobsize: '65535'
        maxconnections: '1024'
        binlogdir: /var/lib/beanstalkd/binlog
        binlogfsync: null
        binlogsize: '10485760'
    beanstalk_console: 0
    binlogdir: /var/lib/beanstalkd/binlog
rabbitmq:
    install: '0'
    settings:
        port: '5672'
    users: {  }
    vhosts: {  }
    plugins: {  }
elastic_search:
    install: '0'
    settings:
        version: 1.4.1
        java_install: true
solr:
    install: '0'
    settings:
        version: 4.10.2
        port: '8984'
```

**Comment by jtreminio** [comment]
Make sure no other application on your host machine is using port 80. Skype does this by default.

Semi-related, but I would highly suggest you place any vhosts in a subdirectory of `/var/www`, instead of directly in it.

Example, instead of `www_root: /var/www/public`, do `www_root: /var/www/public/charthamvh.dev/public`.

`/var/www` should be the directory containing all your vhosts.

You also don't seem to have a `public` directory from your `ls` output.

**Comment by squaredindex** [comment]
Thanks jtreminio! I've made the changes you've suggested. Turns out I was getting some build errors that I didn't spot before, this was stopping nginx from completing its install, ergo no way to render to the browser.

However I'm still getting this error: 
![state](https://cloud.githubusercontent.com/assets/7317110/7221218/546fa9fe-e6da-11e4-9c0d-df9acb5dffc7.png)
Which seems to leave me able to view this at **192.168.57.101**
![current](https://cloud.githubusercontent.com/assets/7317110/7221229/b13d48e4-e6da-11e4-8866-d248345f55b1.png)
Also nothing at my specified domain (charthamvh.dev)
